sale_report_hide_price/tests/test_sale.py

```diff
@@ -9,20 +9,20 @@ def setup(self):
         super(SaleOrderTest, self).setup()
 
     def test_saleorder_action(self):
-        self.sale_order = self.env['sale.order']
+        self.sale_order_obj = self.env['sale.order']
         self.partner = self.env.ref('base.res_partner_2')
         self.products = {
             'serv_order': self.env.ref('product.product_delivery_01'),
             'serv_deli': self.env.ref('product.product_order_01'),
         }
 
-        self.order_idA = self.sale_order.create({
+        self.order_idA = self.sale_order_obj.create({
```

Comment:
@ilakkia-serpentcs  Please update self.sale_order_id instead of  self.order_idA

Reply:
updated self.sale_order_id